ngraph_bridge/ngraph_utils.cc

```diff
@@ -586,6 +589,30 @@ void ClearAttribute(Graph* g,
   }
 }
 
+static ngraph::pass::PassConfig* _p_pass_config = nullptr;
+
+static void InitPassConfig() {
+  // use a singleton config object
+  if (_p_pass_config) return;
+  _p_pass_config = new ngraph::pass::PassConfig();
+  // Set the defaults, unless already specified via env var
+  if (ngraph::getenv_string("NGRAPH_PASS_ENABLES").empty()) {
```

Comment:
This is error-prone. Use `PassConfig` to find out whether a pass was enabled or not.